modules/installation/pages/proxy-setup.adoc

```diff
@@ -207,45 +207,36 @@ filename: "pxelinux.0"
 
 
 
-// REMARK: 2019-08-23, ke: this needs closer checking
-// see https://github.com/SUSE/spacewalk/issues/10521
-// and https://github.com/uyuni-project/uyuni-docs/issues/110
-[[replacing.a.susemgrproxy]]
+[[replace-susemgrproxy]]
 == Replace a {susemgrproxy}
 
-A {susemgrproxy} is dumb in that it does not contain any information about the clients that are connected to it.
-A {susemgrproxy} can therefore be replaced by a new one.
-Naturally, the replacement proxy must have the same name and IP address as its predecessor.
+A proxy does not contain any information about the clients that are connected to it.
+Therefore, a proxy can be replaced by a new one at any time.
+The replacement proxy must have the same name and IP address as its predecessor.
 
-In order to replace a {susemgrproxy} and keeping the clients registered to the proxy leave the old proxy in {productname}.
+Leave the old proxy in place while you prepare the replacement.
```

Comment:
Maybe it should be clarified what "leave in place" means. Of course it should be shut down because the new proxy will have the same IP address as the old one and having two machines with the same IP is not really fun :)

Reply:
I push a change for clarity.